blocks/init/src/Blocks/manifest.json

```diff
@@ -1,8 +1,11 @@
 {
-	"$schema": "https://raw.githubusercontent.com/infinum/eightshift-frontend-libs/develop/schemas/settings.json",
+	"$schema": "https://raw.githubusercontent.com/infinum/eightshift-frontend-libs/develop/schemas/globalManifest.json",
 	"namespace": "eightshift-boilerplate",
 	"background": "#FBF9FF",
 	"foreground": "#9973E3",
+	"config": {
+		"outputVariablesInline": true
```

Comment:
Maybe we should change the key name to `outputCssVariablesGlobally` or reverse the logic of the current one, that makes a tad more sense (inline would mean that they’re _inline_ with the components) 😄

Reply:
I agree with this. First part that I've noticed was the missing `Css` in the name, and the second one was the inline part that confused me (that's what we're doing by default).

Reply:
agreed i will put `outputCssVariablesGlobally` instead

Reply:
updated